client/modules/IDE/components/FloatingActionButton.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import PlayIcon from '../../../images/triangle-arrow-right.svg';
+import StopIcon from '../../../images/stop.svg';
+import { prop, remSize } from '../../../theme';
+import { startSketch, stopSketch } from '../actions/ide';
+
+const Button = styled.div`
+  position: fixed;
+  right: ${remSize(20)};
+  bottom: ${remSize(20)};
+  height: ${remSize(60)};
+  width: ${remSize(60)};
+  z-index: 3;
+  cursor: pointer;
+  ${prop('Button.secondary.default')};
+  aspect-ratio: 1/1;
```

Comment:
This property [has way more support than I thought](https://caniuse.com/mdn-css_properties_aspect-ratio)!  It's not really necessary since you've defined both a `width` and a `height` but it's not hurting anything so 🤷 .

Reply:
yeah 😂😂